src/NServiceBus.RabbitMQ/Routing/DirectRoutingTopology.cs

```diff
@@ -4,6 +4,7 @@
     using System.Collections.Generic;
     using System.Linq;
     using global::RabbitMQ.Client;
+    using Logging;
```

Comment:
This should have been removed by the second commit (which should be squashed before merging).